cmd/kusk/cmd/deploy.go

```diff
@@ -57,11 +57,12 @@ func init() {
 	//add to root command
 	rootCmd.AddCommand(deployCmd)
 
-	deployCmd.Flags().StringVarP(&file, "file", "f", "", "file to deploy")
+	deployCmd.Flags().StringVarP(&file, "in", "i", "", "file path or URL to OpenAPI spec file to generate mappings from. e.g. --in apispec.yaml")
```

Comment:
Does it actually work when you supply a URL? @mortada-codes just opened an issue that he is unable to apply an API from a URL

Reply:
nope :)
Good that you reminded me. On it

Reply:
I take it back it will work with URLs 
```
# go run cmd/kusk/main.go deploy -i https://raw.githubusercontent.com/aabedraba/kusk-demo/main/openapi.yaml --name test   
api.gateway.kusk.io/test created
```